pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "evergreen-lint"
-version = "0.1.3"
+version = "0.1.4"
```

Comment:
I believe there is one other place where the version is stored. I think it's in the root level `__init__.py`. Please update that as well. The constant is called `__version__`